### PR TITLE
feat(styles): add skeleton examples and styling [ci visual]

### PR DESCRIPTION
**Pull request by droshev**
## Related Issue
Closes none

## Description
add skeleton examples and styling

### CODE REVIEW

File: packages/styles/src/fundamental-styles.scss [hunk 1]
```diff
@@ -108,3 +108,4 @@
 
 // BTP Specific components/overwrites
 @import "./btp/avatar.scss";
+@import "skeleton";
```
Comment (InnaAtanasova) [inline packages/styles/src/fundamental-styles.scss:111]: why is this here? Under line 109 are only BTP components overwrites aka have the same names as Fiori

File: packages/styles/src/skeleton.scss [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    fill: var(--sapContent_Placeholderloading_Background);
+  }
+
+  &__middle {
+    stop-color: var(--fdSkeletonGradientMiddleColor);
+    animation: fd-skeleton-shimmer 2s linear infinite;
+  }
+
+  &--animated {
+    .#{$block}__canvas {
```
Comment (InnaAtanasova) [inline packages/styles/src/skeleton.scss:24]: I don't get this specificity, `--fdSkeletonGradientMiddleColor` is part of `fd-skeleton__middle` element where you specify the stop color and the animation. Please check.

File: packages/styles/src/skeleton.scss [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    display: block;
+    fill: var(--sapContent_Placeholderloading_Background);
+  }
+
+  &__middle {
+    stop-color: var(--fdSkeletonGradientMiddleColor);
+    animation: fd-skeleton-shimmer 2s linear infinite;
+  }
+
+  &--animated {
```
Comment (InnaAtanasova) [inline packages/styles/src/skeleton.scss:23]: I don't get this modifier class. Shouldn't the `fd-skeleton` be always animated? I couldn't find an example with just `fd-skeleton` class. It's always `<div class="fd-skeleton fd-skeleton--animated" style="...">`.
In this case do we need this extra modifier?

File: packages/styles/stories/Components/skeleton/skeleton.example.html [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+            </linearGradient>
+            <mask id="fd-skeleton-0">
+                <rect x="0" y="0" width="100%" rx="4" height="100%" class="ng-star-inserted"></rect>
+            </mask>
+        </defs>
+        <rect x="0" y="0" width="100%" height="100%" mask="url(#fd-skeleton-0)">
+    
+        </rect>
+    </svg>
+</div>
```
Comment (InnaAtanasova) [inline packages/styles/stories/Components/skeleton/skeleton.example.html:24]: please add empty line at the end of file

**Review by InnaAtanasova — CHANGES_REQUESTED** [review]

File: packages/styles/stories/Components/skeleton/circle.example.html [hunk 5]
```diff
@@ -0,0 +1,17 @@
+<div class="fd-skeleton fd-skeleton--animated" style="width: 50px; height: 50px;">
+    <svg width="100%" height="100%" class="fd-skeleton__canvas">
+        <defs>
+            <linearGradient id="skeletonGradient">
+                <stop offset="0" stop-color="var(--sapContent_Placeholderloading_Background)"></stop>
+                <stop class="fd-skeleton__middle">
+                    <animate attributeName="offset" values="0; 0; 0.5; 1; 1" dur="2s" repeatCount="indefinite"></animate>
+                </stop>
+                <stop offset="1" stop-color="var(--sapContent_Placeholderloading_Background)"></stop>
```
Comment (InnaAtanasova) [inline packages/styles/stories/Components/skeleton/circle.example.html:9]: for me this is very weird. Why do you set the `stop-color` for the first and last stop in the markup and for the middle you use a class?

File: packages/styles/src/skeleton.scss (hunk 2 above)
Comment (InnaAtanasova) [reply packages/styles/src/skeleton.scss:24]: ok, I got it :) It's because you have `<stop offset="0" stop-color="var(--sapContent_Placeholderloading_Background)"></stop>` which is a child element of `fd-skeleton__canvas`. Weird approach but I understand now. I would still use a class for the forst and last stop (like for the middle).

File: packages/styles/src/fundamental-styles.scss (hunk 1 above)
Comment (droshev) [reply packages/styles/src/fundamental-styles.scss:111]: this was autogenerated

File: packages/styles/src/skeleton.scss (hunk 3 above)
Comment (droshev) [reply packages/styles/src/skeleton.scss:23]: merged it with the block

File: packages/styles/stories/Components/skeleton/circle.example.html (hunk 5 above)
Comment (droshev) [reply packages/styles/stories/Components/skeleton/circle.example.html:9]: added a class for it

File: packages/styles/src/skeleton.scss [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+  cursor: wait;
+
+  &__canvas {
+    display: block;
+    fill: url(#skeletonGradient);
+
+    --fdSkeletonGradientMiddleColor: #999;
+  }
+
+  &__begin-end {
```
Comment (InnaAtanasova) [inline packages/styles/src/skeleton.scss:20]: maybe rename to `&__outer` ?

File: packages/styles/stories/Components/skeleton/circle.example.html [hunk 7]
```diff
@@ -0,0 +1,17 @@
+<div class="fd-skeleton" style="width: 50px; height: 50px;">
+    <svg width="100%" height="100%" class="fd-skeleton__canvas">
+        <defs>
+            <linearGradient id="skeletonGradient">
+                <stop offset="0" stop-color="var(--sapContent_Placeholderloading_Background)"></stop>
```
Comment (InnaAtanasova) [inline packages/styles/stories/Components/skeleton/circle.example.html:5]: shouldn't this be a class now?

File: packages/styles/stories/Components/skeleton/circle.example.html [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+                <stop class="fd-skeleton__middle">
+                    <animate attributeName="offset" values="0; 0; 0.5; 1; 1" dur="2s" repeatCount="indefinite"></animate>
+                </stop>
+                <stop offset="1" stop-color="var(--sapContent_Placeholderloading_Background)"></stop>
```
Comment (InnaAtanasova) [inline packages/styles/stories/Components/skeleton/circle.example.html:9]: same. Shouldn't this be a class now?

**Review by InnaAtanasova — CHANGES_REQUESTED** [review]
Please update all examples to use classes not inline var for the stop color

**Review by InnaAtanasova — APPROVED** [review]
